scripts/download-analyzers.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
I get this error:
`$ sh download-analyzers.sh`
`download-analyzers.sh: 3: set: Illegal option -o pipefail`

Reply:
Yes, the script requires bash, as indicated by the first line. You can run
with `bash download-analyzers.sh` or `./download-analyzers.sh`

Reply:
Could we output a warning in that case?

Reply:
We could. But I would rather not, because I don't think it's a common practice.

Note that in the README I wrote to run this with `./scripts/download-analyzers.sh`, and that way it should work fine, or raise an error if the user doesn't have Bash installed.